tensorflow_probability/python/distributions/ordered_logistic.py

```diff
@@ -246,6 +271,10 @@ def _augmented_cutpoints(self):
         tf.constant(np.inf, dtype=cutpoints.dtype))
     return tf.concat([-inf, cutpoints, inf], axis=-1)
 
+  def _augmented_log_survival_function(self):
```

Comment:
Probably can inline this in both places, since it is very short.

Reply:
done in https://github.com/tensorflow/probability/pull/795/commits/1f1727eb121074b6e4c2d53546c910e3895c5f32